monitoring/messagesigning/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# start by pulling the python image
```

Comment:
By moving your code to the mock_uss module, this file will not be required anymore. 
Instead, create a `run_locally_[SERVICE_NAME].sh` similar to `[run_locally_scdsc.sh](https://github.com/interuss/dss/blob/master/monitoring/mock_uss/run_locally_scdsc.sh)`.

Based on the design document requested in #893, we will be able to agree on the actual SERVICE_NAME and if we need a new script or should update the existing one.

For your information, the docker image which will be used is defined in [`monitoring/Dockerfile`](https://github.com/interuss/dss/blob/master/monitoring/Dockerfile)
Please do not modify the docker image itself. 
If you need to use the keys defined in the configuration below, please use a volume mount in the shell script like [this](https://github.com/interuss/dss/blob/master/monitoring/mock_uss/run_locally_scdsc.sh#L30)